### DISCUSSION TITLE
Downloaded videos are lower quality than originals

### DISCUSSION THREAD

**Discussion by gc435** [aclap-dev/video-downloadhelper#63] [Help and Q&A] [answered]
### Your question

I'm a new VDH user and license owner.

My downloaded videos seem to be lower quality than the originals.  The resolution seems fine, but the frame rate does not look good.  However, I confirmed the downloaded videos are 24 fps.  I am no expert, so perhaps the perceived lower quality is from some other reason.  I notice this in videos where people speak.  Their lips move more smoothly in the original videos.  Audio seems to be synced OK though.

VDH says the source's bitrate is 84400.  When downloaded, Windows File Explorer Properties says the Data Rate is 611 kbps and the Total Bitrate is 864 kbps.  I don't know if this is relevant.

Both the source and downloaded videos are 720 x 480.

Any ideas?

### Addon & CoApp version and details

version: 8.1.4.0
beta: false
build date: 2024-01-04
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: accepted
key (only first 16 fisrt characters): [redacted]
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:122.0) Gecko/20100101 Firefox/122.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.10",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\[redacted]",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "6.0"
}

**Comment by jcfrog** [comment, marked as answer]
Hi,
Would you mind trying with the latest beta release? https://www.downloadhelper.net/beta

**Reply by GrampaWildWilly** [reply]
Also, the latest CoApp: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-Installation

Also, post some URLs of videos you believe are not as good as they should be.  By that I mean simply the URLs of pages on which the videos reside, not anything more complicated.

**Reply by gc435** [reply]
I tried again using **8.2.0.19a1 (beta)**, and I think the quality is better.

I'm downloading a training video that I paid for, but it's behind a paywall.  The video is a presenter speaking to the viewer.  Before this beta, the downloaded files had perfect audio, but the presenter's lips did not move smoothly.  It's better with the beta.
